lib/fritzbox/smarthome/smoke_detector.rb

```diff
@@ -0,0 +1,21 @@
+module Fritzbox
+  module Smarthome
+    class SmokeDetector < Actor
+
+      attr_accessor \
+        :alert_state,
+        :last_alert
+
+      class << self
+        def new_from_api(data)
+          return if data.dig('alert', 'state') == nil
+          @values = {
+            alert_state:            data.dig('alert', 'state').to_i,
+            last_alert:             Time.at(data.dig('alert', 'lastalertchgtimestamp').to_i)
```

Comment:
💅 Can we reduce the whitespace here?

```suggestion
            alert_state: data.dig('alert', 'state').to_i,
            last_alert:  Time.at(data.dig('alert', 'lastalertchgtimestamp').to_i)
```